test/manual/tls_support.test.ts

```diff
@@ -106,6 +108,42 @@ describe('TLS Support', function () {
       });
     });
   });
+
+  context('when tlsCertificateKeyFile is provided, but tlsCAFile is missing', () => {
```

Comment:
Is this test timing out after 10 seconds because we are expecting a server selection timeout error after 30?

Reply:
Yes, I think so. Is the option that governs that `connectTimeoutMS`?

Reply:
Nevermind, I need both `connectTimeoutMS` and `serverSelectionTimeoutMS`